app/layout.tsx

```diff
@@ -0,0 +1,26 @@
+import './global.scss';
+
+export const metadata = {
+  title: 'UpLeveled - stripe',
+  icons: {
+    shortcut: '/images/logo.png',
+  },
+};
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <html lang="ENG">
```

Comment:
this seems like a wrong value for `lang`, is `ENG` correct?

Reply:
https://github.com/upleveled/connecting-to-external-apis-secrets-with-stripe-fall-2022-vienna-austria/blob/85f6653d67eeb9ccaf099006071fdc8d8db4124b/app/layout.tsx#L17